instrumentation/opentelemetry-instrumentation-aiohttp-server/tests/test_aiohttp_server_integration.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import pytest_asyncio
+import aiohttp
+from http import HTTPStatus
+from .utils import HTTPMethod
+from pkg_resources import iter_entry_points
```

Comment:
Entrypoint loading has been updated to not use pkg_resources https://github.com/open-telemetry/opentelemetry-python/blob/0fa0ce56ea5cd3f59e7f17ce62f33e49be01dc06/opentelemetry-api/src/opentelemetry/util/_providers.py#L45. I know this is just a test but I believe there were some performance issues with pkg_resources. Can you update here as well?

Reply:
Updated.